src/test/java/com/zendesk/maxwell/row/RowMapTest.java

```diff
@@ -55,4 +56,198 @@ public void testTimestampConversion() throws Exception {
 		int ts = (int) output.get("ts");
 		Assert.assertEquals(ts, timestampSeconds);
 	}
+
+	@Test
+	public void testPkToJsonHash() throws IOException {
+
+		long timestampMillis = 1496712943447L;
+
+		List<String> pKeys = new ArrayList<>();
+
+		pKeys.add("id");
+
+		pKeys.add("name");
+
+		Position position = new Position(new BinlogPosition(1L, "binlog-0001"), 0L);
```

Comment:
It seems like timestampMillis and position are used in each test, and it doesn't look like you care about their values. In which case you might as well hoist them to a field instead of setting them in each test.